pkgs/development/python-modules/uni-curses/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, ncurses, x256, setuptools }:
+
+buildPythonPackage rec {
+  pname = "unicurses";
+  version = "2.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "Uni-Curses";
+    sha256 = "sha256-Anz5YMZULdMhZ802T3tj9LOcBZZfUML1fPOJbC47nVA=";
+  };
+
+  buildInputs = [ ncurses ];
+  propagatedBuildInputs = [ x256 ];
+
+  postPatch = ''
+    substituteInPlace './unicurses/__init__.py' \
+      --replace "find_library('ncursesw')" '"${ncurses}/lib/libncursesw.so.6"' \
+      --replace "find_library('panelw')" '"${ncurses}/lib/libpanelw.so.6"'
+  '';
+
```

Comment:
Are there tests? If not add doCheck = false with a short comment.

Reply:
I believe unicurses does have tests, but tuifi does not.